modules/Bio/EnsEMBL/Variation/BaseTranscriptVariation.pm

```diff
@@ -283,6 +395,63 @@ sub translation_end {
     return $self->{translation_end};
 }
 
+=head2 translation_start_unshifted
```

Comment:
In some cases the sub also sets the translation_end_unshifted. Description suggests only gets/sets unshifted start position. A similar coment applies to  
sub cdna_start 
sub cdna_start_unshifted 
sub cds_start 
sub cds_start_unshifted 
sub translation_start